packages/terra-alert/docs/MoreInformation.md

```diff
@@ -0,0 +1,142 @@
+      <Button onClick={this.handleShowMore} text="Show More" variant="link" size="tiny" />
+    </Toggle>
+    <Toggle isOpen={this.state.allTextShown} isAnimated>
+        Now we are engaged in a great civil war, testing whether that nation, or any nation so conceived and so dedicated, can long endure. We are met on a great battle-field of that war. We have come to dedicate a portion of that field, as a final resting place for those who here gave their lives that that nation might live. It is altogether fitting and proper that we should do this.
+      <br /><br />
+        But, in a larger sense, we can not dedicate -- we can not consecrate -- we can not hallow -- this ground. The brave men, living and dead, who struggled here, have consecrated it, far above our poor power to add or detract. The world will little note, nor long remember what we say here, but it can never forget what they did here. It is for us the living, rather, to be dedicated here to the unfinished work which they who fought here have thus far so nobly advanced. It is rather for us to be here dedicated to the great task remaining before us -- that from these honored dead we take increased devotion to that cause for which they gave the last full measure of devotion -- that we here highly resolve that these dead shall not have died in vain -- that this nation, under God, shall have a new birth of freedom -- and that government of the people, by the people, for the people, shall not perish from the earth.
+      <br /><br />
+      <Button onClick={this.handleShowLess} text="Show Less" variant="link" size="tiny" />
+    </Toggle>
+    </span>
+  </span>
+</Alert>
+```
+![Image](images/longText1.png?raw=true)
+![Image](images/longText2.png?raw=true)
+__________
+Alert component of type 'warning' with a custom Action button.
+```jsx
+<Alert type={Alert.Opts.Types.WARNING} 
+       action={<Button text="Action" size="medium" variant="primary" onClick={actionFunc} />} >
+  This is a warning. It is configured with a custom Action button.
+</Alert>
+```
+![Image](images/actionButton.png?raw=true)
+__________
+Alert component of type 'success' that is dismissible.
+```jsx
+<Alert type={Alert.Opts.Types.SUCCESS} onDismiss={this.handleDismiss} >
+  This is a success alert. It is configured to be dismissible. Click on the Dismiss button to dismiss the alert.
+</Alert>
+```
+![Image](images/dismissible.png?raw=true)
+__________
+Alert of type 'custom' with no icon or title that is dismissible and has an action button
+```jsx
+<Alert type={Alert.Opts.Types.CUSTOM} 
+       onDismiss={this.handleDismiss} 
+       customStatusColor="orange" 
+       action={<Button text="Action" size="medium" variant="primary" onClick={this.actionFunc} />} >
+  This is a custom alert with no icon or title. It is configured to be dismissible 
+  and with a custom action button. Click on the Dismiss button to dismiss the alert.
+</Alert>
+```
+![Image](images/actionAndDismissible.png?raw=true)
+__________
```

Comment:
Possible Bug: is the Action button not getting displayed? 
Not seeing it here or in the "Build your own Alert" section.